src/BubbleChart/BubbleChart.xml

```diff
@@ -76,6 +76,16 @@
                         <attributeType name="Integer"/>
                     </attributeTypes>
                 </property>
+                <property key="autoBubbleSize" type="boolean" defaultValue="true">
+                    <caption>Auto bubble size</caption>
+                    <category>Data points</category>
+                    <description/>
+                </property>
+                <property key="markerSizeReference" type="integer" defaultValue="0">
+                    <caption>Marker size reference</caption>
+                    <category>Data points</category>
+                    <description>The percentage to be used if the Auto bubble size is false</description>
```

Comment:
`markerSizeReference` should be used also for the auto.

 Description:
`When 'Auto bubble size' is true: The scale represent the percentage of the largest bubble. The largest bubble will cover the 10% of the canvas when the scale is set to 10.
When 'Auto bubble size' is false: The 'Bubble size' value is represented as bubble pixel width when the scale is set to 100 percent`

Reply:
`When 'Auto bubble size' is true: The largest bubble will cover the 10% of the of the chart when the scale is set to 10. When 'Auto bubble size' is false: The 'Bubble size' is in pixels width multiplied by the scale percentage`

Reply:
When 'Auto bubble size' is true, the largest bubble will cover the specified percentage of the chart. When 'Auto bubble size' is false, the 'Bubble size' is the specified percentage of the width in pixels.

Reply:
`The percentage bubble size. When 'Auto bubble size' is true, the largest bubble will cover the specified percentage of the chart. When 'Auto bubble size' is false, the bubble size is the pixel value of the 'Bubble size attribute' multiplied by the percentage of the scale.`